articles/storage/files/storage-files-identity-ad-ds-update-password.md

```diff
@@ -15,11 +15,41 @@ If you registered the Active Directory Domain Services (AD DS) identity/account
 
 To trigger password rotation, you can run the `Update-AzStorageAccountADObjectPassword` command from the [AzFilesHybrid module](https://github.com/Azure-Samples/azure-files-samples/releases). This command must be run in an on-premises AD DS-joined environment using a hybrid user with owner permission to the storage account and AD DS permissions to change the password of the identity representing the storage account. The command performs actions similar to storage account key rotation. Specifically, it gets the second Kerberos key of the storage account, and uses it to update the password of the registered account in AD DS. Then, it regenerates the target Kerberos key of the storage account, and updates the password of the registered account in AD DS. You must run this command in an on-premises AD DS-joined environment.
 
+To prevent password rotation, during the onboarding of the Azure Storage account in the domain, make sure to place the Azure Storage Account into a separate organizational unit in AD DS. Disable Group Policy inheritance on this organizational unit to prevent default domain policies or specific password policies to be applied.
```

Comment:
Hello @yannickdils , thanks for helping improve our documentation, we appreciate it!

I showed your script changes to the engineering team and they decided we will stay with the original script content. However, this paragraph they were okay with adding. If you'd like to remove your script changes from this PR, we can integrate it. Or I can make my own PR where I take this paragraph.

Reply:
Hi @roygara, love to give back to the community!

Ok perfect, If it's ok for you I'll remove the script changes.

Reply:
@roygara  I've reverted the script changes. Should be good to go now.